src/Analyzer.Utilities/DiagnosticHelpers.cs

```diff
@@ -16,6 +16,22 @@ public static class DiagnosticHelpers
             DiagnosticSeverity.Warning;
 #endif
 
+        public const bool EnabledByDefaultIfNotBuildingVSIX =
+#if USE_INTERNAL_IOPERATION_APIS // Building Analyzer VSIX
```

Comment:
Why do we have different defaults for VSIX vs build? If it's noisy it's noisy - why does build have more\less tolerance than editing in VS?